domains/anna.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "annanas271",
+           "email": "iamrvins@gmail.com",
+           "discord": "1206164385911799850"
+        },
+    
+        "record": {
+            "CNAME": "annanas271.github.io"
```

Comment:
I cannot reach your website
`annanas271.github.io` redirects me to `miaroschi.rocks` with returns
> This site can’t be reached
> Check if there is a typo in miaroshi.rocks.
> DNS_PROBE_FINISHED_NXDOMAIN